code/game/objects/items/weapons/clown_items.dm

```diff
@@ -134,4 +134,27 @@
 	desc = "Damn son, where'd you find this?"
 	icon_state = "air_horn"
 	honksound = 'sound/items/AirHorn2.ogg'
-	cooldowntime = 50
\ No newline at end of file
+	cooldowntime = 50
+
+/obj/item/weapon/bikehorn/golden
+	name = "golden bike horn"
+	desc = "Golden? Clearly, its made with bananium! Honk!"
+	icon_state = "gold_horn"
+	item_state = "gold_horn"
+
+/obj/item/weapon/bikehorn/golden/attack(mob/living/carbon/M, mob/living/carbon/user)
+	flip_mobs()
+	return ..()
+
+/obj/item/weapon/bikehorn/golden/attack_self(mob/living/carbon/M, mob/user)
+	flip_mobs()
+	return ..()
+
+/obj/item/weapon/bikehorn/golden/proc/flip_mobs(mob/living/carbon/M, mob/user)
+	if (!spam_flag)
+		for(M in ohearers(7, user))

```

Comment:
use the turf of the horn instead of user

var/turf/T = get_turf(src)

ohearers(7, T)

unless you want the user to flip as well, then juse use hearers